packages/textmate-grammars/README.md

```diff
@@ -1,8 +1,95 @@
 # Theia - Textmate Grammars
 
-Contributes basic textmate grammars and language configuration.
-See [here](https://github.com/theia-ide/theia) for a detailed documentation.
+Contributes basic textmate grammars and language configuration. See
+[here](https://github.com/theia-ide/theia) for a detailed documentation.
```

Comment:
Side note: I always find this line weird.  It links to some more generic documentation about Theia as a whole, not more detailed.  The detailed doc is in the packages' doc. Anyway.

Reply:
It should link to https://www.theia-ide.org/doc/index.html